R/pkg_check.R

```diff
@@ -22,21 +22,14 @@ recipes_pkg_check <- function(pkg = NULL, ...) {
     }
   }
   if (any(!good)) {
-    pkList <- paste(pkg[!good], collapse = ", ")
-    msg <- paste0(
-      sum(!good),
-      ifelse(sum(!good) > 1, " packages are", " package is"),
-      " needed for this step and",
-      ifelse(sum(!good) > 1, " are", " is"),
-      " not installed. (",
-      pkList,
-      "). ",
-      "Start a clean R session then run: "
-    )
-    cat(msg)
-
     install_opt <- quos(...)
     install_pkg <- pkg[!good]
+    n_install_pkgs <- sum(!good)
+
+    cli::cli_text(
+      "{n_install_pkgs} package{?s} ({.pkg {install_pkg}}) {?is/are} needed for this step but {?is/are} not installed."
```

Comment:
```suggestion
      "{n_install_pkgs} package{?s} ({.pkg {install_pkg}}) {?is/are} needed ",
      "for this step but {?is/are} not installed."
```

we can split up this string to keep it inside 80 chars